code/dqc/symfilegrowth.q

```diff
@@ -0,0 +1,16 @@
+\d .dqc
+
+/- Take the average symfile count from the past n days, then check that todays
+/- sym file count hasn't grown more than pct%.
+symfilegrowth: {[directory;ndays;pct]
+  /- Get handles to the DQE and the DQEDB. 
+  h:first exec w from .servers.SERVERS where proctype=`dqedb;
+  / Make sure we have enough days in the dqedb.
+  if[ndays>h"count .Q.pv";:(0b;"ERROR: number of days (ndays) exceeds number of available dates")];
+  / Get todays sym file count.
+  tc:dqeh"first exec resvalue from .dqe.resultstab where funct=`symcount";
```

Comment:
Is this used elsewhere (`dqeh`)? If the DQE restarts will this handle go stale? There are TorQ functions you could use here instead that should ensure you don't need to store a handle that could possibly go stale.

Reply:
I'll change it. Which torq functions?

Reply:
Used .servers.getservers, good suggestion from daniel